apps/src/applab/applab.js

```diff
@@ -1453,3 +1453,34 @@ Applab.getAssetDropdown = function (typeFilter) {
   });
   return options;
 };
+
+/**
+ * Return droplet dropdown options representing a list of ids currently present
+ * in the DOM, optionally limiting the result to a certain HTML element tagName.
+ * @param {undefined|string} tagFilter Optional HTML element tagName to filter for.
```

Comment:
Actually believe, "?string" indicates a nullable type rather than an optional parameter. (http://usejsdoc.org/tags-type.html)

Either of the following can be used to indicate an optional argument
@param {string} [tagFilter]
@param {string=} tagFilter

The first [brackets] is jsdoc style, the second is closure style. We should probably standardize on one or the other.  I saw us using the square backet style somewhere else.